supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -323,13 +323,13 @@ There is no functional difference between the first server that was installed an
 *See also*:
 
 [[ingress]]
-==== image:images/yes.png[yes] Ingress (noun)
-*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The Ingress resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
+==== image:images/yes.png[yes] ingress (noun) (adjective)
```

Comment:
```suggestion
==== image:images/yes.png[yes] ingress (noun, adjective)
```

```suggestion
==== image:images/yes.png[yes] ingress
```

It seems odd to me to have two sets of parentheses. I suggest either putting both parts of speech inside a single set of parentheses, or omit the parts of speech altogether. Or split them into two entries, one for noun and one for adjective.

Reply:
Thanks @bredamc , sorry for being so delayed getting back to this! Initially, I created a separate adjective entry, but initial reviews found that more confusing. So I added the adjective ingress definition to the noun ingress definition. I think I'll just leave it as is, and add the comma between the parts of speech.